ads/_config.js

```diff
@@ -119,6 +120,7 @@ export const adPreconnect = {
     'https://eu1.wbdds.com',
   ],
   gmossp: 'https://cdn.gmossp-sp.jp',
+  yieldbot: 'https://cdn.yldbt.com/js/yieldbot.intent.js',
```

Comment:
This would be origins only, but no need to repeat the origin from the preconnect.